grails-app/controllers/org/pih/warehouse/api/CombinedShipmentItemApiController.groovy

```diff
@@ -60,21 +60,21 @@ class CombinedShipmentItemApiController {
 
     def findOrderItems() {
         List<Order> orders
-        if (params.orderIds) {
-            orders = Order.findAllByIdInList(params.orderIds)
+        if (request.JSON.orderIds) {
```

Comment:
Larger topic that needs to be added to our tech debt / tech huddle topics would be how we want to handle params <=> request.JSON in a backwards compatible way.

https://github.com/openboxes/openboxes/labels/tech-huddle.